test/spec/functions-openrosa.spec.js

```diff
@@ -366,8 +366,11 @@ describe( 'Custom "OpenRosa" functions', () => {
     } );
 
     it( 'uuid()', () => {
-        const result = g.doc.evaluate( 'uuid()', g.doc, null, g.win.XPathResult.STRING_TYPE );
+        let result = g.doc.evaluate( 'uuid()', g.doc, null, g.win.XPathResult.STRING_TYPE );
         expect( result.stringValue ).to.have.length( 36 );
+
+        result = g.doc.evaluate( 'uuid(6)', g.doc, null, g.win.XPathResult.STRING_TYPE );
+        expect( result.stringValue ).to.have.length( 6 );
```

Comment:
There is no particular format requirement, so this should be good! Fyi, this is the ODK Collect code: https://github.com/opendatakit/javarosa/blob/59611cc0755dfbaf6631866ff15d6a9a516df704/src/main/java/org/javarosa/core/util/PropertyUtils.java#L54